packages/ics721/src/state.rs

```diff
@@ -38,6 +38,8 @@ pub const INCOMING_CLASS_TOKEN_TO_CHANNEL: Map<(ClassId, TokenId), String> = Map
 /// is `None`) is stored in this map. When the token is returned to
 /// it's source chain, the metadata is removed from the map.
 pub const TOKEN_METADATA: Map<(ClassId, TokenId), Option<Binary>> = Map::new("j");
+/// The admin address for instantiating new cw721 contracts. In case of None, contract is immutable.
+pub const CW721_ADMIN: Item<Option<Addr>> = Item::new("l");
```

Comment:
Main point of the PR is here

Reply:
Per discussion, this name doesn't really describe what's going on.  Please rename to something like `ADMIN_USED_FOR_CW721`. 

The point of this store is to *copy the admin here when instantiating new cw721 contracts